CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 Version 0.2.0-dev (Changes since version 0.2.0 go here)
 -------------------------------------------------------
 
+* Users can now update and extend sample templates by using the "Update sample templates" button, rather than having to do 2 steps with the previous design.
```

Comment:
Are these really 3 new features? Perhaps a way to rephrase it is:
* The extent and update in templates (prep and sample) have been merged. Now you can do both via the GUI by clicking the Update button.